### DISCUSSION TITLE
Can I use the native Android operating system via IKev2 for client connection?

### DISCUSSION THREAD

**Discussion by Nikolaevdo** [strongswan/strongswan#1860] [Q&A]
Hello!

Why do all sources on the Internet write that connecting Android to the strongswan server via IKev2 is possible through a third-party client on Google play?
Is it possible to use built-in tools in Android to enable client connection over IKev2?

After all, this protocol is in Android
I am using Strongswan Linux strongSwan U5.9.9/K6.1.18-100.fc36.x86_64

Phone Oneplus 5T
Android 10
I have 3 types of Ikev2 to choose from:
IPsec Xuath PSK
IPsec Xuath RSA
IPsec Hybrid RSA

**Comment by TelDragon** [comment]
I also have the same question
[https://github.com/strongswan/strongswan/discussions/1843](https://github.com/strongswan/strongswan/discussions/1843)

**Comment by Nikolaevdo** [comment]
Here is my ike sethings:
Here is my file for Ike
[ike.txt](https://github.com/strongswan/strongswan/files/12450641/ike.txt)

**Comment by tobiasbrunner** [comment]
> I have 3 types of Ikev2 to choose from:
> IPsec Xuath PSK
> IPsec Xuath RSA
> IPsec Hybrid RSA

None of these are IKEv2 authentication methods. They sound like IKEv1 methods.
